addon/helpers/invoke.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+const { Helper: { helper }, get, set } = Ember;
+
+/*
+ * This method invokes the named object's method, so what?
+ *
+ *
+ *
+ */
+export function invoke([fun, ...args]) {
+  var obj = args.pop();
+  if(Ember.isArray(obj)){
```

Comment:
Ember.RSVP.all should be Destructured to all only? 

-- 
Fabio Mazarotto

On 19 de março de 2016 at 05:55:26, Lauren Tan (notifications@github.com) wrote:

In addon/helpers/invoke.js:

> @@ -0,0 +1,25 @@
> +import Ember from 'ember';
> +
> +const { Helper: { helper }, get, set } = Ember;
> +
> +/*
> + * This method invokes the named object's method, so what?
> + *
> + *
> + *
> + */
> +export function invoke([fun, ...args]) {
> +  var obj = args.pop();
> +  if(Ember.isArray(obj)){
Destructure isArray

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub